### PR TITLE
Handle events query change

### PR DESCRIPTION
**Pull request by henryzbw**

### CODE REVIEW

File: src/graphql/schema.js [hunk 1]
```diff
@@ -291,6 +298,8 @@ type Query {
     skip: Int
     pendingTxsAddress: String
     includeRoundBets: Boolean
+    userAddress: String
```
Comment (dwalintukan) [inline src/graphql/schema.js]: lets name this something more descriptive: `roundBetsAddress`

File: src/graphql/resolvers.js [hunk 2]
```diff
@@ -70,29 +70,58 @@ module.exports = {
           eventRound: currentRound,
         });
 
+
         // Sum all bets by index
         const rounds = fill(Array(numOfResults), '0');
+        const userRound = fill(Array(numOfResults), '0');
         each(bets, (bet) => {
+          if (userAddress && userAddress.toLowerCase() === bet.betterAddress) {
```
Comment (dwalintukan) [inline src/graphql/resolvers.js]: i created a helper `toLowerCase` method in `utils/index.js`. please use that one.

File: src/graphql/resolvers.js [hunk 3]
```diff
@@ -70,29 +70,58 @@ module.exports = {
...
+        const userBetRound = fill(Array(numOfResults), '0');
+        if (includeBetRoundBets) {
+          const bets = await DBHelper.findBet({
+            txStatus: TX_STATUS.SUCCESS,
+            eventAddress: address,
+            eventRound: 0,
+          });
+
+          each(bets, (bet) => {
+            if (userAddress && userAddress.toLowerCase() === bet.betterAddress) {
```
Comment (dwalintukan) [inline src/graphql/resolvers.js]: same, use helper toLowerCase

File: src/graphql/resolvers.js (hunk 2 above)
Comment (dwalintukan) [reply src/graphql/resolvers.js]: and technically wont need to check if `userAddress` is defined. `bet.betterAddress` should always be defined. so if userAddress was undefined, youd be checking `undef === bet.betterAddress` which will fail anyways.

File: src/graphql/resolvers.js [hunk 4]
```diff
@@ -70,29 +70,58 @@ module.exports = {
...
         // Add result set amount if round 1
         if (currentRound === 1) {
           const resultSet = await DBHelper.findOneResultSet({
             txStatus: TX_STATUS.SUCCESS,
             eventAddress: address,
             eventRound: 0,
           });
+
           if (!isNull(resultSet)) {
+            if (userAddress && userAddress.toLowerCase()  === resultSet.centralizedOracleAddress) {
```
Comment (dwalintukan) [inline src/graphql/resolvers.js]: use toLowerCase helper

File: src/graphql/resolvers.js [hunk 5]
```diff
@@ -70,29 +70,58 @@ module.exports = {
...
+            }
             rounds[resultSet.resultIndex] = sumBN(
               rounds[resultSet.resultIndex],
               resultSet.amount,
             ).toString(10);
           }
         }
-
-        return rounds;
+        return {totalRoundBets: rounds, userRoundBets: userRound, totalBetRoundBets: betRound, userBetRoundBets: userBetRound};
```
Comment (dwalintukan) [inline src/graphql/resolvers.js]: nit: place key/value on separate lines

**Comment by henryzbw** [comment]
> Ship after changes

need to go with frontend, otherwise may cause some error.

**Pull request merged**